Tests/SecureXPCTests/Client & Server/Server Error Handler Test.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  Server Error Handler Test.swift
+//  
+//
+//  Created by Josh Kaplan on 2022-02-23.
+//
+
+import Foundation
+import XCTest
+import SecureXPC
+
+// Note: it's intentional this is *not* `Codable` to validate that server side all errors are sent to the error handler
+private enum ExampleError: Error, Equatable {
+    case completeAndUtterFailure
+}
+
+class ServerErrorHandlerTest: XCTestCase {
+    
+    func testErrorHandler_Sync()  throws {
+        let errorToThrow = ExampleError.completeAndUtterFailure
+        let errorExpectation = self.expectation(description: "\(errorToThrow) should be provided to error handler")
```

Comment:
Even if you're not using async code in the SUT, you can use it in your test here to remove the need for expectations.

https://www.swiftbysundell.com/articles/unit-testing-code-that-uses-async-await/#an-alternative-to-expectations

Reply:
NVM, I see why it's not so straight forward in this case. See my comments below.